packages/commonwealth/server/routes/createChain.ts

```diff
@@ -481,6 +482,8 @@ const createChain = async (
     // });
   }
 
+  getChainActivity.queryWithCacheOverride(models);
```

Comment:
@jnaviask do you thin we should wait for query to finish?
- it will take second or two extra, thanks!

Reply:
added await